File: BryceResortPatrol/ClientApp/src/app/models/candidate.ts
export class Candidate {
  firstName: string;
  lastName: string;
  email: string;
  phoneNumber: string;
  description: string;
  constructor(
    firstName: string,
    lastName: string,
    email: string,
    phoneNumber: string,
    description: string) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.email = email;
    this.phoneNumber = phoneNumber;
    this.description = description;
  }
}

File: BryceResortPatrol.Common/Models/Candidate.cs
﻿namespace BryceResortPatrol.Common.Models;

public record Candidate
{
    public string Id { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Email { get; set; }
    public string PhoneNumber { get; set; }
    public string Description { get; set; }

    public override string ToString()
    {
        return @$"
                Id: {Id}, 
                FirstName: {FirstName},
                LastName: {LastName},
                Email: {Email},
                PhoneNumber: {PhoneNumber},
                Description: {Description}";
    }
}
